### DISCUSSION TITLE
Option to have dungeon openings unlocked in single player?

### DISCUSSION THREAD

**Discussion by jkmcdani** [diasurgical/DevilutionX#7135] [Ideas]
Hello,

I have a request regarding single player mode for diablo classic.  Can you have a gameplay menu option where the dungeon entrances are unlocked in town automatically with the level restrictions from multiplayer?  

I would like to be able to play single player and be able to quickly run through the levels 13-16 without having to work all the way down through the dungeon.

Thanks.

**Comment by Trihedraf** [comment]
Duplicate of #6910

This idea has been brought up before and was decided against changing.

**Reply by jkmcdani** [reply]
Thanks.  I wish you all would reconsider.  Even if it was only after one has beaten hell difficulty.  I'd really like to be able to go straight into Hell in Hell difficulty without having to teleport down.

Thanks for putting together the mod.

**Reply by AJenbo** [reply]
If you play Hellfire it works like this.

**Reply by Chonkblonk** [reply]
Almost, you still need to go dig up the map in the Hive to open up the Crypt every time, at least in Single Player.
Personally, I think making this a toggle wouldn't be THAT out of line with various other tweaks and options which are offered.

**Reply by StephenCWills** [reply]
You don't need the map in DevX if you're high enough clvl.

**Reply by Chonkblonk** [reply]
My single player Barbarian is level 39 and he still needs to go fetch the map. Are you thinking of multiplayer?
![bild](https://github.com/user-attachments/assets/6a39fc6f-612d-4183-a5be-40fac85b8fe7)

**Reply by StephenCWills** [reply]
I'm not sure what I was thinking of. I thought we added some logic to open the grave based on clvl, but I'm not finding it in the code now.

**Reply by Chonkblonk** [reply]
I've been in the same boat of being certain that I implemented something, yet actually never did.

**Reply by AJenbo** [reply]
> I'm not sure what I was thinking of. I thought we added some logic to open the grave based on clvl, but I'm not finding it in the code now.

I made the reset of them more consistent, but specifically didn't do that one because it's normally done via a quest.

**Reply by Chonkblonk** [reply]
How about a toggle for enabling it if your character is high enough level?

**Comment by kphoenix137** [comment]
> Hello,
> 
> I have a request regarding single player mode for diablo classic. Can you have a gameplay menu option where the dungeon entrances are unlocked in town automatically with the level restrictions from multiplayer?
> 
> I would like to be able to play single player and be able to quickly run through the levels 13-16 without having to work all the way down through the dungeon.
> 
> Thanks.

Multiplayer now has single player quests, so the differences between single player and multiplayer now are not that extreme, although monsters have improved stats in multiplayer (It's still not that hard though, single player is easy mode basically). Multiplayer also doesn't require that you have publicly running games either. It has these town warps that can be accessed at all times as long as you meet the level requirement. Might be worth doing it that way.

**Reply by Chonkblonk** [reply]
Agree.
